next/components/ui/FormRJSF/GeneratedFormRJSF.tsx

```diff
@@ -20,6 +22,13 @@ interface FormRJSF {
 const GeneratedFormRJSF = ({ eform, escapedSlug, formSlug }: FormRJSF) => {
   const [t] = useTranslation('forms')
   const form = useFormStepper(escapedSlug, eform.schema)
+  const [isOnShowSkipModal, setIsOnShowSkipModal] = useState<boolean>(false)
+  const [skipModalWasShowed, setSkipModalWasShowed] = useState<boolean>(false)
```

Comment:
```suggestion
  const [skipModalWasShown, setSkipModalWasShown] = useState<boolean>(false)
```

please rename elsewhere as well